pkg/crc/machine/machine.go

```diff
@@ -553,6 +557,7 @@ func Status(statusConfig ClusterStatusConfig) (ClusterStatusResult, error) {
 	defer libMachineAPIClient.Close()
 
 	openshiftStatus := "Stopped"
+	podmanStatus := "service available"
```

Comment:
Maybe we want to use something more consistent the openshift state? `Podman service: Running`?